checkov/terraform/evaluation/evaluation_methods/const_variable_evaluation.py

```diff
@@ -43,6 +46,11 @@ def _assign_definition_value(self, definition_type, var_name, var_value, var_ass
                 definition_path = assignment_obj.get('definition_path')
                 entry_expression = assignment_obj.get('definition_expression')
                 definition_name = assignment_obj.get('definition_name')
```

Comment:
Isn't `local.ip_ranges.ipv4Prefixes[*].prefix` a string?

Reply:
@tronxd It is evaluated as a tree
![image](https://user-images.githubusercontent.com/16308767/83740062-0a1d0780-a65f-11ea-8442-2eb359be4486.png)
Unlike regular variables:
![image](https://user-images.githubusercontent.com/16308767/83740118-17d28d00-a65f-11ea-9945-3817c2da1bd7.png)